README.md

```diff
@@ -1,20 +1,22 @@
 ## packer-builder-triton
 
-[HashiCorp Packer](https://packer.io) builder for [Joyent Triton](https://www.joyent.com) or private installations of [Smart Datacenter](https://www.joyent.com/private-cloud).
+[HashiCorp Packer](https://packer.io) builder for
+[Joyent Triton](https://www.joyent.com) or private installations of
+[Smart Datacenter](https://www.joyent.com/private-cloud).
```

Comment:
Open source Triton is still Triton. "SmartDataCenter" and "sdc" are historical names.